python/setup.cfg

```diff
@@ -54,4 +54,4 @@ packages = find:
 install_requires =
     numpy
     numba>=0.49
-python_requires = >=3.7,<3.9
+python_requires = >=3.8
```

Comment:
```suggestion
python_requires = >=3.8, <3.10
```

Reply:
Hmm based on [this thread](https://github.com/conda-forge/staged-recipes/pull/17986#issuecomment-1040591473) should we be avoiding this constraint altogether? @leofang @jakirkham